### PR TITLE
Use FQCN instead of short names for modules

### PR DESCRIPTION
**Pull request by redhatrises**
- Recommended by Ansible to avoid using external collections that might have the same name

### CODE REVIEW

**Comment by redhatrises** [comment]
Please don't merge until we have molecule merged as I want to test these changes against it.

**Comment by redhatrises** [comment]
Ready for review @carlosmmatos

**Review by carlosmmatos — APPROVED** [review]
lgtm

**Pull request merged**
